docker/Dockerfile.agent

```diff
@@ -3,8 +3,8 @@ FROM python:3.9
 COPY README.md pyproject.toml poetry.lock /src/
 COPY beamer /src/beamer
 
-RUN mkdir -p ./beamer/data/relayers
```

Comment:
I'm not sure why this changed, but OK

Reply:
The thing is, we use `WORKDIR` to change the working directory to `/src` and run  poetry from there, so the agent will expect to find relayers there, not under `/beamer/...`.